tools/sigmac/logsource-mapping.yml

```diff
@@ -0,0 +1,235 @@
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 5
+    # Driver Loaded: Sysmon 6
+    driver_load:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 6
+    # Image (DLL, etc...) Loaded: Sysmon 7
+    image_load:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 7
+    # Create Remote Thread: Sysmon 8
+    create_remote_thread:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 8
+    # Raw Access Thread: Sysmon 9
+    raw_access_thread:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 9
+    # Process Access: Sysmon 10
+    process_access:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 10
+    # File Creation: Sysmon 11
+    file_event:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 11
+    # Registry Add: Sysmon 12, Security 4657
+    registry_add:
+        Types: 2
+        Type-1:
+            Channel: 'Microsoft-Windows-Sysmon/Operational'
+            EID: 12
+        Type-2:
+            Channel: 'Security'
+            EID: 4657
+    # Registry Delete: Sysmon 12
+    registry_delete:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 12
+    # Registry Set: Sysmon 13, Security 4657
+    registry_set:
+        Types: 2
+        Type-1:
+            Channel: 'Microsoft-Windows-Sysmon/Operational'
+            EID: 13
+        Type-2:
+            Channel: 'Security'
+            EID: 4657
+    # Registry Rename: Sysmon 14
+    registry_rename:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 14
+    # Registry Event: Sysmon 12-14, Security 4657
+    registry_event:
+        Types: 2
+        Type-1:
+            Channel: 'Microsoft-Windows-Sysmon/Operational'
+            EID:
+                - 12
+                - 13
+                - 14
+        Type-2:
+            Channel: 'Security'
+            EID: 4657
+    # Create Stream Hash: Sysmon 15
+    create_stream_hash:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 15
+    # Pipe Created: Sysmon 17 & 18
+    pipe_created:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID:
+            - 17
+            - 18
+    # WMI Event: Sysmon 19-21
+    wmi_event:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID:
+            - 19
+            - 20
+            - 21
+    # DNS Query: Sysmon 22
+    dns_query:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID: 22
+    # File Delete: Sysmon 23 & 26
+    file_delete:
+        Types: 1
+        Channel: 'Microsoft-Windows-Sysmon/Operational'
+        EID:
+            - 23
+            - 26
+    # Clipboard Capture: Sysmon 24
+    create_stream_hash:
```

Comment:
Since create_stream_hash appears in two places, should it be in one place?

Reply:
Yes, it should have been `clipboard_capture`. Thank you! I fixed it and moved the file into the `config` directory as it will eventually be used in hayabusa instead of just the converter.